package/src/lsif.ts

```diff
@@ -19,6 +19,10 @@ function pathFromDoc(doc: sourcegraph.TextDocument): string {
 }
 
 function setPath(doc: sourcegraph.TextDocument, path: string): string {
```

Comment:
It seems somewhat wrong to me that there are `path` values that are actually URIs. How does that happen? Can this be solved in a less confusing way?